### ISSUE TITLE
Allow the setting/overriding of the default theme at the styleguide-wide level

### ISSUE THREAD

**Issue by migbar** [chrislopresto/ember-freestyle#65]
Currently, the `ember-freestyle` service holds the default theme for the styleguide.

It would be nice if we could set a different theme from the `freestyle-guide` component.

Something like:

``` handlebars

{{#freestyle-guide
    title='Ember Freestyle'
    subtitle='Living Style Guide'
    highlightJsTheme='Monokai'
}}

 ...

{{/freestyle-guide}}

```

The `highlightJsTheme` argument will not be mandatory. If not provided, the guide will fall back to the service's `defaultTheme` as it does today.

However, if a theme is given, it will become the `defaultTheme` for the guide.

We should preserve the current flexibility of overriding the `defaultTheme` at the `freestyle-usage` component.

In other words, we should still be able to do

``` handlebars

{{#freestyle-usage 'foo-normal' title='Normal' highlightJsTheme='tomorrow'}}
  {{x-foo status='normal'}}
{{/freestyle-usage}}

```

which would override the theme for that instance of the `freestyle-usage` component, but not clobber the overall theme of the guide.
#### Possible benefits:

We can then implement a theme choice from the settings (or some other place)

**Comment by chrislopresto** [comment]
Just realized this issue was still open. This is now resolved by your work on https://github.com/chrislopresto/ember-freestyle/pull/104

Thanks again, @migbar!
